readme.md

```diff
@@ -81,7 +81,8 @@ Your Rails database has two tables. `students` has the columns `id` and `name`,
 Use ActiveRecord to create a new `pbj` sandwich and make it belong to the student named Geraldo.
 
 ```rb
-# Your answer...
+create.Student(id: 0, name: "Geraldo")
+create.Sandwich(id: 0, type: "pbj", student_id: 0)
 
 ```
 
```

Comment:
You should not be assigning `id` values yourself. Rails takes care of that automatically.

You're writing out Active Record commands in a reversed way. It should actually look something like this...

```ruby
geraldo = Student.create( name: "Geraldo" )
pbj = geraldo.sandwiches.create( type: "Peanut Butter & Jelly" )
```